src/main/java/meditracker/argument/IntakeFrequencyArgument.java

```diff
@@ -1,15 +1,21 @@
 package meditracker.argument;
 
 /**
- * Intake frequency of medication
+ * Represents the intake frequency of medication.
+ * Extends the Argument class.
  */
 public class IntakeFrequencyArgument extends Argument {
```

Comment:
Sry! I might not have been clear earlier. 
If user can specify the dosage to be taken at each time (morning, afternoon and evening), then I think that would already imply the intakeFreq in a single day. Yes?

Otherwise, when we have them together, u might have to include a new check to see when user specify -dM -dA, then the intakeFreq should be 2. It wouldnt make sense if user accidentally specify intakeFreq other than 2. 
I just think this will introduce extra redundant steps for u.

Reply:
@e0958902 Can take a look into this as well, perhaps remove the dosage argument (since we have dosage morning, afternoon, evening) and the intake frequency argument (cus of what @T0nyLin mentioned above)

Reply:
Yes, do remove the dosage argument as we are doing a daily basis tracking(with three time slots) where there is a same amount of dosage everyday. But morning dose may not be the same as afternoon or evening.